.github/workflows/build-and-push.yml

```diff
@@ -3,133 +3,64 @@ on:
   push:
     branches:
       - master
+  schedule:
+      - cron: '0 1 * * 3'
 
 jobs:
   build-and-push:
+    if: github.repository_owner == 'sclorg'
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         include:
-          - version: "9.6"
+          - dockerfile_path: "9.6"
```

Comment:
This is obsolete. Version 9.6 reached EOL.

Reply:
I removed it.